### PR TITLE
3426 credentials windows machine

### PR DESCRIPTION
**Pull request by VakarisZ**
# What does this PR do?

Adds packer scripts for windows chrome credentials machine

Add any further explanations here.

## PR Checklist
* [ ] Have you added an explanation of what your changes do and why you'd like to include them?
* [ ] Is the TravisCI build passing?
* [ ] Was the CHANGELOG.md updated to reflect the changes?
* [ ] Was the documentation framework updated to reflect the changes?
* [ ] Have you checked that you haven't introduced any duplicate code?

## Testing Checklist

* [ ] Added relevant unit tests?
* [ ] Do all unit tests pass?
* [ ] Do all end-to-end tests pass?
* [ ] Any other testing performed?
    > Tested by {Running the Monkey locally with relevant config/running Island/...}
* [ ] If applicable, add screenshots or log transcripts of the feature working

### CODE REVIEW

File: envs/monkey_zoo/packer/setup_rdp_64.yml [hunk 1]
```diff
@@ -18,10 +18,6 @@
           - Administrators
           - "Remote Desktop Users"
 
-    - name: Allow port 8080
-      win_command:
-        cmd: netsh advfirewall firewall add rule name="Allow Port 8080" dir=in action=allow protocol=TCP localport=8080
-
```
Comment (VakarisZ) [inline envs/monkey_zoo/packer/setup_rdp_64.yml:24]: Revert and add a comment that this is used as tunneling port

File: envs/monkey_zoo/packer/setup_rdp_64.yml [hunk 2]
```diff
@@ -18,10 +18,6 @@
           - Administrators
           - "Remote Desktop Users"
 
-    - name: Allow port 8080
-      win_command:
-        cmd: netsh advfirewall firewall add rule name="Allow Port 8080" dir=in action=allow protocol=TCP localport=8080
```
Comment (ilija-lazoroski) [inline envs/monkey_zoo/packer/setup_rdp_64.yml:23]: We need 8080 on 64, so 65 can communicate back if I am not wrong.

File: envs/monkey_zoo/packer/files/chrome-creds.ps1 [hunk 3]
```diff
@@ -0,0 +1,339 @@
...
+			$res = $cmd.ExecuteNonQuery()
+
+			$cmd.dispose()
+
+			Write-Host "...all credentials deleted!"
+
+		}
+}
+
+#$conn.Close()
```
Comment (ilija-lazoroski) [inline envs/monkey_zoo/packer/files/chrome-creds.ps1:339]: ?

File: envs/monkey_zoo/packer/files/chrome-creds.ps1 [hunk 4]
```diff
@@ -0,0 +1,339 @@
...
+			$param = $cmd.Parameters.AddWithValue("@password_type", $password_type)
+			$param = $cmd.Parameters.AddWithValue("@times_used", $times_userd)
+			$param = $cmd.Parameters.AddWithValue("@form_data", (HexToByteArray($form_data)))
+			$param = $cmd.Parameters.AddWithValue("@display_name", $display_name)
+			$param = $cmd.Parameters.AddWithValue("@icon_url", $icon_url)
+			$param = $cmd.Parameters.AddWithValue("@federation_url", $federation_url)
+			$param = $cmd.Parameters.AddWithValue("@skip_zero_click", $skip_zero_click)
+			$param = $cmd.Parameters.AddWithValue("@generation_upload_status", $generation_upload_status)
+			$param = $cmd.Parameters.AddWithValue("@possible_username_pairs", (HexToByteArray($possible_username_pairs)))
+			#$param = $cmd.Parameters.AddWithValue("@id", '9') # AUTO INCREMENT
```
Comment (ilija-lazoroski) [inline envs/monkey_zoo/packer/files/chrome-creds.ps1:308]: ?

File: envs/monkey_zoo/packer/files/chrome-creds.ps1 (hunk 3 above)
Comment (VakarisZ) [reply envs/monkey_zoo/packer/files/chrome-creds.ps1:339]: I've fixed this script to work for our case, don't want to spend time improving readability or whatnot

**Review by ilija-lazoroski — APPROVED** [review]

**Comment by VakarisZ** [comment]
Merged in c5ef012990e6c6596a5f28a7166df883557cc4f3

**Pull request closed without merging**